DataManagementSystem/DB/FileCatalogComponents/FileMetadata.py

```diff
@@ -330,127 +330,98 @@ def __transformMetaParameterToData( self, metaname ):
     result = self.db._update( req )
     return result
 
-  def __createMetaSelection( self, meta, value, table = '' ):
-
-    if type( value ) == types.DictType:
-      selectList = []
+#########################################################################
+#
+#  Finding files by metadata
+#
+#########################################################################
+
+  def __createMetaSelection( self, value ):
+    ''' Create selection string to be used in the SQL query
+    '''
+    queryList = []
+    if type( value ) == types.FloatType:
```

Comment:
Agreed. I think that 
isinstance( value, IntType ) is as good and may be even more explicit.
I was happy to discover that the following also works as expected:
>>> isinstance( 'rrr', StringTypes )
True
>>> isinstance( u'rrr', StringTypes )
True
>>> isinstance( r'rrr', StringTypes )
True
>>> isinstance( 1, (int, long) )
True
>>> isinstance( 1L, (int, long) )